src/handle.rs

```diff
@@ -125,6 +125,11 @@ pub struct Global<T> {
   isolate_handle: IsolateHandle,
 }
 
+
+// This is probably not safe because NonNull is explicitly !Send
+unsafe impl<T> Send for Global<T> where T: Send + Sync {}
+unsafe impl<T> Sync for Global<T> where T: Send + Sync {}
```

Comment:
I suspect we're going to have to do more work to make `Global` actually safe to send across threads. This may involve refactoring Global to replace `NotNull<T>` with something that is `Send + Sync`. Any idea on what can be done here?

Reply:
I think using  a raw pointer or `NonNull` is inevitable. Rather, we have to make sure that whatever you can do with a Global does not violate safety guarantees. Note that anything you do with a Global requires an entered scope (so if we maintain the invariant that entering a scope is safe, we're mostly there). There's also some exception: you can compare a Global to another Global/Local, as well as get its hash, without having entered a scope; I suspect this is where it'll get really difficult.